### ISSUE TITLE
Photos exported from Illustrator wont work?

### ISSUE THREAD

**Issue by IronDrachen** [GrandyB/SheetsIO#60]
Hello, Is there a size limit or specific formatting rules for PNG/JPG files? 

I'm able to attach photos I find online, but I cant seem to get the files I'm exporting as PNG or JPEG with Adobe Illustrator to work. 

_I get this error:_
![Screenshot 2024-09-28 064928](https://github.com/user-attachments/assets/14f2638c-49e3-4a2b-b9ca-7bca3fb064e2)

**_Attached is a link to the logs, the photos I'm trying to use, and the illustrator files I'm exporting from._**  

- PNG

https://drive.google.com/drive/folders/1YAYe7-wldWcfi4O7HDDQbVepVH2ervoP?usp=drive_link

- JPEG

https://drive.google.com/drive/folders/1doKkXbEysbviJ-_MWkJV0Xn2VijSzoXp?usp=drive_link

- Illustrator

https://drive.google.com/drive/folders/1YAkZQl-EDmQo5aQoEVhsq7iBODmPB2Cb?usp=drive_link

- Log

https://drive.google.com/file/d/1kqf4i-nfOLCYAS72Go95_bnchc_U3QVR/view?usp=drive_link

Thanks for any help!

**Comment by GrandyB** [comment]
Hello, just downloaded your 13.png into a local folder, referred to it similarly (`file://D:/13.png`) with config entry `{ "cell": "A3", "name": "test2", "fileExtension": "png" }` and had no error.

I also don't see an error in your log file, just the following:
```DEBUG application.services.FileIO - Treating file://C:/test2/13%2Epng as a local image url```

**Comment by IronDrachen** [comment]
Wow I have no idea why its that big, it should be 1920x1080. Let me check my output settings and try again. 

I'll comment back when I've tried the smaller photos. Thanks so much for answering back

**Comment by IronDrachen** [comment]
I got it! It worked with the smaller files, but I did try the bigger ones again and they work now too. I'm not sure why it was having issues, but it seems to have resolved itself. My machine did install a windows update since I tried, so maybe that affected something?

I've attached a photo of what I'm using your program for. It's a D&D screen for streaming. The photos are for the health bar at the top and all those stats on the page are updating live. My camera is usually in the bottom left. Thanks so much for the help and this awesome program.  :)

![Screenshot 2024-09-28 232422](https://github.com/user-attachments/assets/75cd7dcc-c220-4341-91c2-1db31d1d3afe)
